sanitize-html/sanitize-html.d.ts

```diff
@@ -1,27 +1,65 @@
-﻿// Type definitions for sanitize-html 1.6.0
+﻿// Type definitions for sanitize-html 1.12.0
 // Project: https://github.com/punkave/sanitize-html
-// Definitions by: Rogier Schouten <https://github.com/rogierschouten>
+// Definitions by: Afshin Darian <https://github.com/afshin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
-declare module "sanitize-html" {
+declare function sanitize(dirty: string, options?: sanitize.IOptions): string;
 
-	function sanitizeHtml(s: string, opts?: {
-		allowedTags?: string[];
-		allowedSchemes?: string[];
-		allowedAttributes?: { [index: string]: string[] };
-		allowedClasses?: { [index: string]: string[] };
-		transformTags?: { [index: string]: any };
-		exclusiveFilter?: {
-			[index: string]: (frame: {
-				tag: string;
-				attribs: { [index: string]: string };
-				text: string;
-				tagPosition: number;
-			}) => boolean
-		};
 
-	}): string;
+declare namespace sanitize {
+  export
+  type Attributes = { [attr: string]: string };
 
-	export = sanitizeHtml;
+
+  export
+  type Tag = { tagName: string; attributes: Attributes; };
+
+
+  export
+  type Transformer = (tagName: string, attributes: Attributes) => Tag;
+
+
+  export
+  interface IDefaults {
+    allowedAttributes: { [index: string]: string[] };
+    allowedSchemes: string[];
+    allowedSchemesByTag: { [index: string]: string[] };
+    allowedTags: string[];
+    selfClosing: string[];
+  }
+
+
+  export
+  interface IFrame {
+    tag: string;
+    attributes: { [index: string]: string };
+    text: string;
+    tagPosition: number;
+  }
+
+
+  export
+  interface IOptions {
+    allowedAttributes?: { [index: string]: string[] };
+    allowedClasses?: { [index: string]: string[] };
+    allowedSchemes?: string[];
+    allowedTags?: string[];
+    exclusiveFilter?: (frame: IFrame) => boolean;
+    selfClosing?: string[];
+    transformTags?: { [tagName: string]: string | Transformer };
+  }
+
+
+  export
+  var defaults: IDefaults;
+
+
+  export
```

Comment:
the `export` keyword is superfluous here. all declarations in a .d.ts are implicitly exported. so please remove it.